cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -120,12 +120,14 @@ func (w *wizard) initializeOptions() []wizardOpt {
 }
 
 func (w *wizard) filterOptions(options []wizardOpt) []wizardOpt {
+	isOrchestratorOrRedeemer := w.orchestrator || w.isRedeemer()
```

Comment:
I think that for the sake of consistency, I'd not execute `w.isRedeemer()` (and therefore an HTTP call) here, but rather in the place where `wizard` is initialized.

https://github.com/livepeer/go-livepeer/blob/783bfdb392b114f8e5ddfcc95624ededaea5788d/cmd/livepeer_cli/livepeer_cli.go#L54

Also, an idea, maybe we should just rename `w.orchestrator` => `w.orchestratorOrRedeemer`?

Reply:
Thanks @leszko, I hadn't realised about the `fix #commit` flow, will do that next time.

I've moved the `isRedeemer` call as suggested, but for the rename there are some places that `w.orchestrator` is used where I wasn't sure if it made sense to also do it for a redeemer node: https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_cli/livepeer_cli.go#L85

If you think it makes sense to include redeemers there, then I'll update to `w.orchestratorOrRedeemer` as suggested

Reply:
That's fine what you did. Let's keep it as you coded it! 👍